### ISSUE TITLE
Gun Valkyrie: Major Slowdown when boosting

### ISSUE THREAD

**Issue by banana3333** [xemu-project/xemu#1058]
### Title

https://xemu.app/titles/49470017/#Gunvalkyrie

### Bug Description

Game slows down massively when boosting

### Expected Behavior

It should not slow down. What CPU and/or GPU should I buy to bruteforce this LOVELY game into fullspeed 100% of the time? No this isn't really a bug, I'm just a big dumb idiot and really wanted to ask this on a xemu forum but didn't see one.

### xemu Version

v0.7.32

### System Information

Windows 10
i5-3570
RX 560 4GB

And a Steam Deck

**Comment by vedikasugano** [comment]
I'm kinda late for the party but,  I have the same issue when I use the iGPU with amdgpu and mesa open source driver.

**System Information**
xemu: 0.7.64
OS: Arch Linux
Kernel: 5.18.15-265-tkg-pds
CPU: AMD Ryzen 7 5700G with Radeon Graphics (iGPU/4G)

No issues when I use nVidia 1050ti (nvidia proprietary driver)

**Comment by ehw** [comment]
I'm not having performance issues with the booster, seems to be okay. Although it might be my hardware/drivers...

Version:      0.7.85
Branch:       master
Commit:       065c74a00fe402f65147e9b0d1a1eeac1b4b609f
Date:         Mon Mar  6 21:06:28 UTC 2023

CPU:          AMD Ryzen 7 5800X 8-Core Processor             
OS Platform:  Windows
OS Version:   Windows
Manufacturer: NVIDIA Corporation
GPU Model:    NVIDIA GeForce RTX 3080/PCIe/SSE2
Driver:       4.0.0 NVIDIA 531.29
Shader:       4.00 NVIDIA via Cg compiler

**Comment by vedikasugano** [comment]
Yeah, been a while it was fixed actually. Working great on the 5700G APU.
